node/parachain/src/cli.rs

```diff
@@ -101,14 +101,10 @@ pub struct RunCmd {
 	pub account_id: Option<H160>,
 }
 
-fn parse_h160(i: &str) -> Result<H160, String> {
-	let hex = hex::decode(i).map_err(|e| e.to_string())?;
-	let x = hex.as_slice();
-	if x.len() == 20 {
-		Ok(H160::from_slice(x))
-	} else {
-		Err("invalid length for H160 public key".to_string())
-	}
+fn parse_h160(input: &str) -> Result<H160, String> {
+	input
+		.parse::<H160>()
```

Comment:
I thought this would be a valid way to parse from a string into H160 because of the tests here https://github.com/paritytech/parity-common/blob/master/ethereum-types/src/hash.rs#L139

but I'm getting this error now
```
 Failed to start Moonbeam Test Node.
Command: ../node/standalone/target/release/moonbase-standalone --chain=./moonbeam-test-specs/simple-specs.json --validator --execution=Native --no-telemetry --no-prometheus --manual-seal --account-id=0x1111111111111111111111111111111111111111 --no-grandpa --force-authoring -linfo --port=19931 --rpc-port=19932 --ws-port=19933 --tmp
Logs:
error: Invalid value for '--account-id <account-id>': Failed to parse H160
```

Reply:
You can use the FromStr with H160.

Reply:
parse uses `FromStr` https://doc.rust-lang.org/std/primitive.str.html#method.parse

the error is for `--account-id=0x1111111111111111111111111111111111111111` so maybe I have to remove the `0x` prefix, but they didn't do that in the tests I linked in the first comment

Reply:
I removed the `0x` prefix in TS by `GENESIS_ACCOUNT.substring(2)` so it would pass `--account-id=1111111111111111111111111111111111111111` instead and this worked